src/main/java/address/controller/PersonEditDialogController.java

```diff
@@ -214,6 +224,12 @@ private boolean isInputValid() {
             }
         }
 
+        try {
+            URL url = new URL(webPageField.getText());
+        } catch( MalformedURLException e){
+            errorMessage += "Invalid web page link.\n";
+        }
```

Comment:
We shouldn't be doing this try-catch in too many places. We'll tidy it up later.